libbeat/outputs/elasticsearch/client.go

```diff
@@ -551,17 +551,18 @@ func (client *Client) PublishEvent(data outputs.Data) error {
 func (client *Client) LoadTemplate(templateName string, template map[string]interface{}) error {
 
 	path := "/_template/" + templateName
-	status, _, err := client.request("PUT", path, "", nil, template)
+	return client.LoadJson(path, template)
```

Comment:
should we check for the error here and "print" a template specific error. Now it seems like it only returns the json error which will be printed out later and does not explain to the user that something with the template didn't work.